modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,10 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3644[OSSM-3644] Federation egress-gateway gets the wrong update of network gateway endpoints, causing extra endpoint entries. This has been updated on the server side so it gets the correct network gateway endpoints.
+
+* https://issues.redhat.com/browse/OSSM-3595[OSSM-3595] Sometimes, `istio-cni` fails on RHEL9 because of SELinux where `iptables-restore` was not allowed to open files in `/tmp`. SELinux now passes `iptables-restore` via `stdin`.
```

Comment:
- The bug fix description needs to be rewritten using the guidelines and template in the SSG: "Use past tense for the problem and present tense for the solution". See: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes
- Also, I think that `RHEL9` is not correct. As a product name, it needs to use an attribute -- either `{op-system-base}` or `{op-system-base-full}`